src/main/java/com/juniorstart/juniorstart/service/UserProfileService.java

```diff
@@ -0,0 +1,119 @@
+package com.juniorstart.juniorstart.service;
+
+import com.juniorstart.juniorstart.exception.BadRequestException;
+import com.juniorstart.juniorstart.exception.ResourceNotFoundException;
+import com.juniorstart.juniorstart.model.UserRole;
+import com.juniorstart.juniorstart.model.User;
+import com.juniorstart.juniorstart.model.UserProfile;
+import com.juniorstart.juniorstart.repository.UserDao;
+import com.juniorstart.juniorstart.repository.UserProfileRepository;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.EnumUtils;
+import org.apache.commons.text.WordUtils;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/** Represents an userProfile service.
+ * @author Rafał maduzia
+ * @version 1.0
+ */
+@Service
+@Slf4j
+public class UserProfileService {
+
+    final private UserProfileRepository userProfileRepository;
+    final private UserDao userRepository;
+
+    public UserProfileService(UserDao userDao, UserProfileRepository userProfileRepository) {
+        this.userProfileRepository = userProfileRepository;
+        this.userRepository = userDao;
+    }
+
+
+    //Ignore This, Future for Next task
+  //  Long userId = new getLogedUserId().userID;
+
+    //TODO GONNA WORK ON ADD USER PROFILE ON NEXT TASK
+    public UserProfile addUserProfile(UserProfile userProfile) {
+
+        //Long userId = new getLogedUserId().userID;
+        //Long tmp = userId;
+        Long userId= 5L;
```

Comment:
?

Reply:
Sprawdzałem wyciąganie PrivateID z zalogowanej sesji, jednak w tym tasku doszedłem do wniosku, że nie będzie mi jednak potrzebny stąd ten komentarz w kodzie. Natomiast widzę napisałeś, wyżej o tej adnotacji, to zobaczę na nią.